src/Scanner/TokenArrayScanner.php

```diff
@@ -380,6 +380,14 @@ protected function scan()
             return $infoIndex;
         };
 
+        // ensure php backwards compatibility
+        if (! defined('T_NAME_QUALIFIED')) {
+            define('T_NAME_QUALIFIED', 24000);
+        }
+        if (! defined('T_NAME_FULLY_QUALIFIED')) {
+            define('T_NAME_FULLY_QUALIFIED', 24000);
```

Comment:
Is there any particular value I should give to these constants?
For now I've just copy-pasted the one already present:
https://github.com/laminas/laminas-code/blob/8d74ede0d9b0850b4d7508997de48ed569151821/src/Scanner/ClassScanner.php#L1098-L1100